src/components/ArtWork/component.js

```diff
@@ -3,8 +3,17 @@ import PropTypes from 'prop-types';
 import './ArtWork.css';
 
 const ArtWork = (albumArtwork) => (
-  <div className='album-artwork-container'>
-    <img alt="artwork" className='album-artwork' src={albumArtwork.albumImage} />
+  <div className="album-artwork-container">
+    {albumArtwork.albumImage ? (
+      <img
+        alt="artwork"
+        className="album-artwork"
+        src={albumArtwork.albumImage}
+      />
+    ) : (
+      //basically display nothing
+      <h1></h1>
+    )}
```

Comment:
you should use `null` if you don't want to display anything.

Reply:
Yes I will use `null `  for this pull request